charts/app-config/values-integration.yaml

```diff
@@ -1823,6 +1823,17 @@ govukApplications:
             key: SECRET_KEY_BASE
       - name: USE_TMPDIR_PAGE_CACHE
         value: "true"
+    nginxConfigMap:
+      extraConf: |
+        # 1stline use this URL in their zendesk template
+        location = /static/gov.uk_logotype_crown.png {
+          absolute_redirect off;
+          return 301 /media/5c810ef4ed915d43e50ce260/gov.uk_logotype_crown.png;
+        }
+
+        location /robots.txt {
+          expires 1w;
```

Comment:
Will this have any effect? It looks like we're currently serving robots.txt from Router's nginx.

Reply:
Removed.